package.json

```diff
@@ -92,7 +92,7 @@
     "ts-loader": "^8.0.17",
     "ts-node": "10.9.1",
     "typedoc": "^0.17.7",
-    "typescript": "4.8.4",
+    "typescript": "4.9.5",
```

Comment:
satisfies operator and newer ts stuff

Reply:
Just wondering we have something for this project like dependabot that we use (whether manual or automated), to identify which of these dependencies need updating and when?

Reply:
we have dependabot but not configured or setup really like i did in the other repo